Dockerfile

```diff
@@ -10,4 +10,4 @@ RUN npm install --production
 
 COPY . .
 
-CMD ["node", "app.js"]
```

Comment:
This should target the compiled app.js file. Node can only run Javascript so the Typescript files get compiled into Javascript for Node to use.

Reply:
Fixed it.